wdl/tasks/Pacasus.wdl

```diff
@@ -0,0 +1,180 @@
+version 1.0
+
+import "Utils.wdl" as Utils
+import "Structs.wdl"
+
+workflow Process {
+    input {
+        File reads
+        Int chunk_size_mb
+    }
+
+    call Utils.ConvertReads {
+        input:
+            reads = reads,
+            output_format = "fasta"
+    }
+
+    call SplitReadsByLength {
+        input:
+            reads_fasta = ConvertReads.converted_reads,
+            length = 55000
+    }
+
+    call SplitReads {
+        input:
+            reads_fasta = SplitReadsByLength.shorter_reads,
+            chunk_size_mb = chunk_size_mb
+    }
+
+    scatter (split_read_fasta in SplitReads.split_reads_fasta) {
+        call RemovePalindromes {
+            input:
+                read_fasta = split_read_fasta
+        }
+    }
+
+    call MergeFasta {
+        input:
+            processed_fastas = flatten([RemovePalindromes.processed_fasta, [SplitReadsByLength.longer_reads]])
+    }
+
+    output {
+        File processed_fasta = MergeFasta.processed_fasta
+    }
+}
+
+task SplitReadsByLength {
+    input {
+        File reads_fasta
+        Int length
```

Comment:
To be consistent with our method for overriding task runtime specs, please add the `RuntimeAttr? runtime_attr_override` to all task inputs.